raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -135,18 +137,25 @@ import Spinner from '@/components/icons/Spinner.vue';
   },
   computed: {
     ...mapState(['tokens', 'raidenAccountBalance']),
+    ...mapGetters(['udcToken']),
   },
 })
 export default class Withdrawal extends Mixins(BlockieMixin) {
   tokens!: Tokens;
   raidenAccountBalance!: string;
+  udcToken!: Token;
   balances: Token[] = [];
   loading: boolean = true;
   withdrawing: boolean = false;
   withdraw: Token | null = null;
 
   async mounted() {
     this.balances = await this.$raiden.getRaidenAccountBalances();
+
+    if (this.udcToken.balance.gt(Zero)) {
+      this.balances = uniq([...this.balances].concat(this.udcToken));
```

Comment:
`uniq` must be applied over an array of comparable objects. I'm not sure you are sure the **reference** of the object in the` udcToken` variable would be the same as whatever gets returned by `getRaidenAccountBalances`.
Therefore, you should use `uniqBy`, passing a function like `(token) => token.address` as second param, in order to make the resulting array unique per token addresses.